### PR TITLE
Pipelines: Adding Deployment stage for products and orders

### PR DESCRIPTION
**Pull request by enrique-ayala**

### CODE REVIEW

File: .github/workflows/orders_pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  development:
+    if: startsWith(github.event.ref, 'refs/heads/main')
+    uses: ./.github/workflows/pipeline_template.yaml
+    needs: [run_unit_tests]
+    with: 
+      stack_name: sam-hiberry-app-orders-development
+      sam_deploy_overrides: "ShopifyEventBusName=aws.partner/shopify.com/71471267841/InfinityCloudSource StageName=development"
+      sam_template: src/orders/template.yaml
+      aws_region: us-east-1
+      pipeline_execution_role: arn:aws:iam::937030943129:role/aws-sam-cli-managed-dev-pipel-PipelineExecutionRole-43mYyIsd3U1C
```
Comment (MarcoBurgos) [inline .github/workflows/orders_pipeline.yaml]: is there a way of using variables or even secrests?

File: .github/workflows/products_pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  development:
+    if: startsWith(github.event.ref, 'refs/heads/main')
+    uses: ./.github/workflows/pipeline_template.yaml
+    needs: [run_unit_tests]
+    with: 
+      stack_name: sam-hiberry-app-products-development
+      sam_template: src/products/template.yaml
+      aws_region: us-east-1
+      pipeline_execution_role: arn:aws:iam::937030943129:role/aws-sam-cli-managed-dev-pipel-PipelineExecutionRole-43mYyIsd3U1C
```
Comment (MarcoBurgos) [inline .github/workflows/products_pipeline.yaml]: same here

File: .github/workflows/orders_pipeline.yaml (hunk 1 above)
Comment (infinity-cloud-solutions) [reply .github/workflows/orders_pipeline.yaml]: Changes applied to use variables

File: .github/workflows/products_pipeline.yaml (hunk 2 above)
Comment (infinity-cloud-solutions) [reply .github/workflows/products_pipeline.yaml]: Changes applied to use variables

**Review by MarcoBurgos — APPROVED** [review]

**Pull request merged**